src/components/PlayerBar/UploadButton/UploadButton.js

```diff
@@ -0,0 +1,24 @@
+// ToDo - create some desigen to the button
+
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { clearSong } from '../../../store/actions/songActions';
```

Comment:
by calling this action form the store,
state.song === null
and AppBrowser.js should send the user back to upload song mockup